app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
+  queue_as :default
+
+  def perform(keyword_id)
+    keyword = Keyword.find(keyword_id)
+    html = GoogleSearchService.new(keyword: keyword.keyword).call
+    utf_8_html = html.force_encoding('iso8859-1').encode('utf-8')
+    update_keyword(keyword, utf_8_html)
+  rescue ActiveRecord::RecordNotFound, GoogleSearch::Errors::SearchKeywordError, ActiveRecord::StatementInvalid
+    update_keyword_status keyword, :failed
+  end
+
+  private
+
+  def update_keyword_status(keyword, status)
+    keyword.update(status: status)
+  end
+
+  def update_keyword(keyword, html)
+    keyword.update({ html: html, status: :completed })
```

Comment:
Does it work without `{}`?

```suggestion
    keyword.update(html: html, status: :completed )
```

Reply:
Let's me check it :D

Reply:
It works

Reply:
Fixed in [`36c45ec` (#45)](https://github.com/suho/google-search-ruby/pull/45/commits/36c45ec1188d351e4058ceb33a9673471bc0f433)